src/config/production.js

```diff
@@ -14,7 +14,35 @@ module.exports = {
     minZoom: 2,
     maxZoom: 18,
 
-    initialView: [-18.632, 18.479]
+    initialView: [-18.632, 18.479],
+
+    oamMosaicLayer: {
+      id: "oam-mosaic",
+      name: "OAM Mosaic",
+      url: "https://apps.kontur.io/raster-tiler/oam/mosaic/{z}/{x}/{y}.png"
+    },
+
+    baseLayers: [
+      {
+        id: "oam-base",
+        name: "Mapbox Light",
+        url:
+          "https://api.mapbox.com/styles/v1/openaerialmap/ciyx269by002w2rldex1768f5/tiles/256/{z}/{x}/{y}?access_token=" +
+          "pk.eyJ1Ijoib3BlbmFlcmlhbG1hcCIsImEiOiJjaXl4MjM5c20wMDBmMzNucnZtbnYwZTcxIn0.IKG5flWCS6QfpO3iOdRveg"
+      },
+      {
+        id: "osm",
+        name: "OpenStreetMap (Standard)",
+        url: "http://a.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      },
+      {
+        id: "satellite",
+        name: "Mapbox Satellite",
+        url:
+          "https://api.mapbox.com/styles/v1/openaerialmap/ciyx28hy800362rto0u9x10fv/tiles/256/{z}/{x}/{y}?access_token=" +
+          "pk.eyJ1Ijoib3BlbmFlcmlhbG1hcCIsImEiOiJjaXl4MjM5c20wMDBmMzNucnZtbnYwZTcxIn0.IKG5flWCS6QfpO3iOdRveg"
```

Comment:
we should place this key as an envvar

Reply:
It's already placed without env vars
https://github.com/hotosm/oam-browser/blob/develop/src/config/production.js#L10